kitchen/ph5tostationxml.py

```diff
@@ -440,7 +440,9 @@ def create_obs_channel(self, station_list, deployment, cha_code, loc_code,
     
     def get_response_inv(self, obs_channel):
         das = obs_channel.data_logger.serial_number
-        self.ph5.read_das_t(das, reread=False)
+        start = (obs_channel.start_date - datetime.datetime(1970,1,1)).total_seconds()
+        end = (obs_channel.end_date - datetime.datetime(1970,1,1)).total_seconds()
+        self.ph5.read_das_t(das, start, end,  reread=False)

```

Comment:
Quick side note:

Eventually I think we should consolidate a lot of the commonly used logic, like the code above, from ph5tomsAPI.py, ph5tostationxml.py, and ph5toexml.py into a ph5utils.py module.

By creating a utilities module we avoid maintaining multiple instances of the same code. For example the ph5tostationxml.py and ph5toexml.py modules both use identical copies of the radial and rectangular geographic coordinate methods.